flow/FlowManager.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.8.22;
+
+import { ud21x18, UD21x18 } from "@prb/math/src/UD21x18.sol";
+
+import { ISablierFlow } from "@sablier/flow/src/interfaces/ISablierFlow.sol";
+
+contract FlowManager {
```

Comment:
Lets add `create` as well. Without it, all the txns would revert since this contract may not be the `msg.sender`.

IIUC, is the objective to introduce a manager that acts on behalf of the users?

Reply:
> Lets add create as well. Without it, all the txns would revert since this contract may not be the msg.sender.

i'm not sure i understand. you can set the `sender` to any address (including the flow manager contract).

> IIUC, is the objective to introduce a manager that acts on behalf of the users?

my rational was to have:
1. a stream creator contract (initiates NULL --> ACTIVE)
2. a stream manager contract (which acts only on ACTIVE streams)
 
does that make sense?

Reply:
what we can do, to be more precise, is to rename the contract to `FlowStreamManager`

Reply:
164a940c5e32c690fbc8cdb2dd6d1f32728acbe2

Reply:
hmm. I didn't get your rationale. `FlowStreamCreator` creates stream with itself as the `sender`. `FlowStreamManager` operates on stream created with itself as the `sender`. The former does not contain other functions so its not useful on its own whereas latter is useful if a stream is created with `FlowStreamManager` as `sender`. 

I think `FlowStreamManager` inheriting from `FlowStreamCreator` would be a good idea.

Reply:
these are just examples, they are not meant to be used anywhere. just demonstrative purposes

>  think FlowStreamManager inheriting from FlowStreamCreator would be a good idea

feel free to push a commit

Reply:
OK fair enough. I agree with you.